extensions/sql-migration/src/models/loginMigrationModel.ts

```diff
@@ -179,7 +180,25 @@ export class LoginMigrationModel {
 			}
 		}
 
-		this.errorCountMap.set(LoginMigrationStep[step], errorBuckets);
+		// Making a string of the map elements of errorBuckets
+		var errorBucketsString = JSON.stringify(
+			Array.from(errorBuckets.entries()).reduce((o: any, [key, value]) => {
+				o[key] = value;
+				return o;
+			}, {})
+		);
+
+		// Retaining this step in case a revert is needed, but we will be using errorCountMapString for telemetry
+		this.errorCountMap.set(LoginMigrationStep[step], errorBucketsString);
+
+		// Creating an array of the error codes and its counts for each step
+		this.errorCountMapString.push(
```

Comment:
Yeah I was experimenting with stringify of the array, but it changes the format with some unnecessary escape characters that I am not able to get rid of: [MigrateLogins:{}],[EstablishUserMapping:{\\"DatabaseNotFoundOnDestinationException\\":1}],[MigrateServerRolesAndSetPermissions:{\\"RoleAlreadyExistsException\\":1}]. So tried to avoid using stringify just to get it in the format we need.